.github/workflows/pr.yml

```diff
@@ -21,10 +21,6 @@ jobs:
       - name: Install php
         if: matrix.os == 'macos-latest'
         run: brew install php
-      - name: Set up Ruby
```

Comment:
This could be removed in the other GH action workflows as well right?

Reply:
Yes, good point